clkRouteTemplate_BUFGCE_X0Y58.txt

```diff
@@ -0,0 +1,82 @@
+int_bufg
```

Comment:
This is not a DCP file, instead it's something routing specific. Please include it with the test.

Reply:
As per https://github.com/Xilinx/RapidWright/pull/229/commits/d04afd8c2c4aaabcb33a20335f6da81f5ba8cd62#r734079887, perhaps this file is not necessary any more?